staging/src/k8s.io/apiserver/pkg/endpoints/metrics/metrics.go

```diff
@@ -185,6 +184,14 @@ var (
 		},
 		[]string{"verb", "group", "version", "resource", "subresource", "scope", "component", "code"},
 	)
+	// RequestAbortsTotal is a number of aborted requests with http.ErrAbortHandler
+	RequestAbortsTotal = compbasemetrics.NewCounter(
```

Comment:
we need to define some labels on it because aborted request won't end up in the audit logs, see `TestErrConnKilledWithAuditFilter` test.

would it be okay to have `verb, group, version and the resource` ?

Reply:
yeah, would add the same we have for other request metrics.